client/nui/app.js

```diff
@@ -43,6 +43,28 @@ function close() {
 	}
 }
 
+function copyToClipboard(text) {
+	text = text.replace(/"/g, '');
+	text = text.trim();
+	var textarea = document.createElement('textarea');
+	textarea.value = text;
+	textarea.style.position = 'absolute';
+	textarea.style.left = '-9999px';
+	document.body.appendChild(textarea);
+	textarea.select();
+	try {
+	  document.execCommand('copy');
+	} catch (err) {
+	  console.error('Unable to copy to clipboard:', err);
+	}
+	document.body.removeChild(textarea);
+}
+
+function toggleflash(status)
+{
+	$('#flashstatus').html(status);
+}
```

Comment:
Refactor this code to alter some already defined markup in the HTML instead of injecting HTML directly. This is very bad practice if you don't know what the user will be sending to you.